action.yml

```diff
@@ -203,15 +213,15 @@ runs:
         ls -l '${{ steps.cerbero-config.outputs.cerbero-home }}'
         echo "::endgroup::"
 
-    - id: bootstrap-cerbero
+    - id: cerbero-build
       working-directory: cerbero
       shell: ${{ runner.os == 'Windows'  && 'msys2 -eo pipefail {0}' || 'bash' }}
       env:
         CI_PROJECT_NAME: cerbero
         LOG_PATH: ${{ steps.cerbero-config.outputs.full-log-path }}
         FETCH_ARGS: --jobs=4
       run: |
-        # Bootstrap Cerbero
+        # Bootstrap Cerbero Log
```

Comment:
Because of the way GH Actions display logs in workflow runs, these comments aren't for the next line, but a description of the whole job step:
<img width="748" alt="image" src="https://github.com/user-attachments/assets/53630386-9c87-4e43-9611-5b3f15fd253d" />

```suggestion
        # Bootstrap Cerbero
```